sdk/groups.go

```diff
@@ -0,0 +1,82 @@
+package sdk
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	apiError "github.com/ONSdigital/dp-identity-api/sdk/errors"
+)
+
+type sortParam string
+
+var SortCreated sortParam = "created"
+var SortName sortParam = "name"
+var SortNameAsc sortParam = "name:asc"
+var SortNameDesc sortParam = "name:desc"
```

Comment:
My preferences (so you can ignore these) are:

```suggestion
var (
  SortCreated sortParam = "created"
  SortName sortParam = "name"
  SortNameAsc sortParam = "name:asc"
  SortNameDesc sortParam = "name:desc"
)
```

I prefer `var` blocks, and vars should follow all the `type` definitions